terragrunt/env/dev/domain/terragrunt.hcl

```diff
@@ -0,0 +1,8 @@
+
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "../../../aws//domain"
```

Comment:
Extra slash here?

```
//domain
```

Reply:
This is how it works. 

https://github.com/cds-snc/covid-alert-metrics-terraform/blob/main/env/staging/s3/terragrunt.hcl

Reply:
You would think that, but it's one of the oddities of Terragrunt that allows it to resolve relative paths.